integration-tests/gradle/src/test/java/io/quarkus/gradle/FastJarFormatWorksTest.java

```diff
@@ -46,22 +46,9 @@ public void testFastJarFormatWorks() throws Exception {
 
             String logs = FileUtils.readFileToString(output, "UTF-8");
 
-            assertThatOutputWorksCorrectly(logs);
-
-            // test that the application name and version are properly set
-            assertThat(DevModeTestUtils.getHttpResponse("/hello", () -> {
-                return jarRun == null ? null : jarRun.isDone() ? "jar run mode has terminated" : null;
-            }).equals("hello"));
```

Comment:
What was the reason to remove the endpoint check? Looks like this is what is actually verifying that the app responds.

Reply:
re-added (earlier)